tests/test_buffer.zunit

```diff
@@ -38,6 +38,8 @@
 }
 
 @test 'ysu - _write_ysu_buffer invalid' {
+    (( $+commands[tput] )) || skip "tput not found"
```

Comment:
I'm not sure about this one. I would want the CI/CD pipeline to fail if `tput` is not installed rather than fail silently.

Reply:
That's understandable. I can drop the `tests: skip tests requiring tput if not present` commit if you prefer. I added this because [YSU gracefully handles a missing `tput`](https://github.com/MichaelAquilina/zsh-you-should-use/blob/f13d39a1ae84219e4ee14e77d31bb774c91f2fe3/you-should-use.plugin.zsh#L5-L7).